examples/generate.sh

```diff
@@ -112,9 +112,13 @@ if [[ ${CAPI_BRANCH} == "stable" ]]; then
   echo "Downloaded ${COMPONENTS_CLUSTER_API_GENERATED_FILE} from cluster-api stable branch - v0.2.5"
 elif [[ ${CAPI_BRANCH} == "dev" ]]; then
   kustomize build ../cluster-api/config/default > "${COMPONENTS_CLUSTER_API_GENERATED_FILE}"
+  kustomize build ../cluster-api/bootstrap/kubeadm/config/default > "${COMPONENTS_CLUSTER_API_GENERATED_FILE}"
+  kustomize build ../cluster-api/controlplane/kubeadm/config/default > "${COMPONENTS_CLUSTER_API_GENERATED_FILE}"
   echo "Generated ${COMPONENTS_CLUSTER_API_GENERATED_FILE} from cluster-api - ${CAPI_BRANCH}"
 else
   kustomize build "github.com/kubernetes-sigs/cluster-api/config/default/?ref=${CAPI_BRANCH}" > "${COMPONENTS_CLUSTER_API_GENERATED_FILE}"
+  kustomize build "github.com/kubernetes-sigs/cluster-api/bootstrap/kubeadm/config/default/?ref=${CAPI_BRANCH}" > "${COMPONENTS_CLUSTER_API_GENERATED_FILE}"
```

Comment:
Do we have any CI setups anywhere that set CAPI_BRANCH to release-0.2 (which would break this)?

Reply:
From https://github.com/kubernetes/test-infra/blob/d3774a6d55ce242179bd29139ae8406f22db729e/config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml CAPI_BRANCH is either set to stable or master